pkg/diskmaker/diskmaker.go

```diff
@@ -132,7 +132,12 @@ func (d *DiskMaker) symLinkDisks(diskConfig DiskConfig) {
 				logrus.Errorf("error creating symlink directory %s with %v", symLinkDirPath, err)
 				continue
 			}
-			symLinkPath := path.Join(symLinkDirPath, deviceNameLoction.diskNamePath)
+			baseDeviceName := filepath.Base(deviceNameLoction.diskNamePath)
```

Comment:
Looks like these are misspelled everywhere so won't hurt, but might want to s/deviceNameLoction/deviceNameLocation/g

Reply:
do you want to sanity check for `.` in the return from filepath.Base incase nothing is provided?  Or is that already covered and I didn't see it?